packages/core/src/RPA/core/locators.py

```diff
@@ -34,6 +35,31 @@ def open_stream(obj, *args, **kwargs):
             obj.close()
 
 
+def find_by_name_or_error(path, criteria):
+    db = LocatorsDatabase(path)
+    if not Path(db.path).exists():
+        db.logger.warning("File does not exist: %s", db.path)
+
+    db.load()
+    if db.error:
+        error_msg, error_args = db.error
+        raise ValueError(error_msg % error_args)
+
+    entry = db.find_by_name(criteria)
+    if not entry:
+        raise ValueError(f"Unknown locator alias: {criteria}")
+
+    if entry["type"] != "browser":
```

Comment:
I don't think this should be here? The library using this can decide if it supports the type or not.

Reply:
Yes absolutely.

Reply:
Fixed at https://github.com/robocorp/rpaframework/pull/58/files